charts/k6-loadtester/Chart.yaml

```diff
@@ -2,12 +2,21 @@ apiVersion: v2
 name: k6-loadtester
 description: Flagger webhook using k6 to do load testing of the canary before rolling out traffic
 type: application
-version: 0.0.1
+version: 0.0.2-pre-01
 appVersion: "0.1.1"
 sources:
   - https://github.com/grafana/flagger-k6-webhook
 keywords:
   - flagger
   - canary
   - webhook
-  - k6
\ No newline at end of file
+  - k6
+home: https://github.com/k8s-at-home/charts/tree/master/charts/stable/autobrr
+icon: https://avatars.githubusercontent.com/u/88781313?s=200&v=4
+maintainers:
+  - name: funkypenguin
+    email: davidy@funkypenguin.co.nz
+annotations:
+  artifacthub.io/changes: |-
+    - kind: added
+      description: Initial release
```

Comment:
Did you copy that over from another repo? The home and icon are wrong. I think having your name as maintainer is misleading as well since you do not have repo access